### DISCUSSION TITLE
Can we generate images like this With latest controlnet at the moment?

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [Mikubill/sd-webui-controlnet#1500] [Q&A]
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/19240467/e6805802-c456-47ef-b597-0585b3bf97f0)

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/19240467/7e6c9c8c-9725-452a-94f8-16a988f14fad)

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/19240467/3aa823bf-be37-4e8d-8f1c-eebcf32cae9f)

**Comment by zixuanz13** [comment]
Yes, you can try depth model. For a detailed tutorial https://stable-diffusion-art.com/text-effect/

**Reply by Njasa2k** [reply]
I think the grapes one is more complicated.
